shared/build.gradle.kts

```diff
@@ -1,51 +1,63 @@
-import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 
 plugins {
-    kotlin("multiplatform")
-    id("com.android.library")
+    alias(libs.plugins.kotlinMultiplatform)
+    alias(libs.plugins.androidLibrary)
+    kotlin("plugin.serialization").version("1.9.22")
+    id("app.cash.sqldelight").version("2.0.1")
```

Comment:
We should move these to the version catalogs to match how other plugins are applied. We can also add them at the root `build.gradle` too, with `apply false` like the other plugins, so that they're loaded there and then used here.